### PR TITLE
Initial recipe for presto immune repertiore sequencing toolkit.

### PR DESCRIPTION
**Pull request by tfenne**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Adds a new recipe for [presto](https://presto.readthedocs.io), a toolkit for immune repertoire sequencing analysis.  This is my first time building a conda recipe.  I've tried my best to read and follow all the instructions, but am happy to make changes if/where I've messed up!

### CODE REVIEW

File: recipes/presto/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+build:
+  number: 0
+  # Presto only supports python3
+  skip: True # [not py3k]
+
+
+requirements:
+  build:
+    - python >=3
```
Comment (bgruening) [inline recipes/presto/meta.yaml]: The pinning here ">=3" is not needed and can be removed.

File: recipes/presto/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - setuptools
+    - numpy >=1.8
+    - scipy >=0.14
+    - pandas >=0.15
+    - biopython >=1.65
+
+  run:
+    - python >=3
```
Comment (bgruening) [inline recipes/presto/meta.yaml]: Here as well

**Comment by tfenne** [comment]
Thanks @bgruening !  I'll make those changes and re-push once all the tests pass on the first go through travis, just in case something fails and I need to make other changes.

**Comment by tfenne** [comment]
Ok, changes made and travis build is passing.  @bgruening are you able to help me with a squash-and-merge?  Thanks!

**Comment by bgruening** [comment]
Sure! Thanks a lot for the quick fixes!

**Pull request merged**